### PR TITLE
feat(data-loader): support specify character encoding when import

### PR DESCRIPTION
**Pull request by kjmnmt**
## Why

Because we want to import CSV files encoded with various encodings.
(Especially, for Shift JIS encoding in the Windows environment.)

## What

- [x] add `encoding` option to import sub-command

## How to test

```
$ yarn build
$ yarn lint && yarn test
```

```
$ yarn build
$ cd packages/data-loader
# Prepare 
$ node lib/main.js import --app <appId> --file-path <path-to-csv-file> --encoding sjis
```

## Checklist

- [x] Read [CONTRIBUTING.md](https://github.com/kintone/js-sdk/blob/master/CONTRIBUTING.md)
- [x] Updated documentation if it is required.
- [x] Added tests if it is required.
- [x] Passed `yarn lint` and `yarn test` on the root directory.

### CODE REVIEW

File: packages/data-loader/src/controllers/import.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import { buildRestAPIClient, RestAPIClientOptions } from "../api";
-import { readFile } from "../utils/file";
+import { ImportEncoding, readFile } from "../utils/file";
```
Comment (ksm-cy) [inline packages/data-loader/src/controllers/import.ts:2]: `ImportEncoding`って名前が動詞に見えてしょうがなくてw（readFileと並んでいるからかな）
`SupportedEncoding`(`SupportedImportEncoding`)などはどうでしょうか？

File: packages/data-loader/src/commands/import.ts [hunk 2]
```diff
@@ -96,6 +104,7 @@ export const handler = (args: Args) => {
     attachmentsDir: args["attachments-dir"],
     filePath: args["file-path"],
     updateKey: args["update-key"],
+    encoding: args.encoding,
```
Comment (ksm-cy) [inline packages/data-loader/src/commands/import.ts:107]: 細かいんですが、他と形式を揃えて`args["encoding"]`にしませんか

File: packages/data-loader/src/utils/file.ts [hunk 3]
```diff
@@ -1,20 +1,27 @@
...
-  const stream = fs.createReadStream(filePath);
+  filePath: string,
+  encoding?: ImportEncoding
+) => Promise<{ content: string; format: string }> = async (
+  filePath,
+  encoding
+) => {
+  const stream = fs
+    .createReadStream(filePath)
+    .pipe(iconv.decodeStream(encoding || "utf8"));
```
Comment (ksm-cy) [inline packages/data-loader/src/utils/file.ts:16]: `encoding`のデフォルト値が設定されているなら、`|| "utf8"`部分がいらなくなります・・？

File: packages/data-loader/src/commands/import.ts (hunk 2 above)
Comment (ksm-cy) [reply packages/data-loader/src/commands/import.ts:107]: Okay, it's ESLint who formats it like this 💯

File: packages/data-loader/src/utils/file.ts (hunk 3 above)
Comment (kjmnmt) [reply packages/data-loader/src/utils/file.ts:16]: @ksm-cy 
Thanks for the comment!
The default with the `--encoding` option is "utf8", but without the option, it is undefined.

**Review by ueokande — APPROVED** [review]

**Pull request merged**